### PR TITLE
Update operatorhub script

### PR DESCRIPTION
**Pull request by rubenvp8510**
Signed-off-by: Ruben Vargas <ruben.vp8510@gmail.com>

## Which problem is this PR solving?
- Update OLM script to make PRs to the correct repositories.

## Short description of the changes
-  Now that the repositories of the operator hub are split in two I updated this script to reflect that fact. Also I replace hub with the official github cli tool.

### CODE REVIEW

File: .ci/operatorhub.sh [hunk 1]
```diff
@@ -21,104 +27,45 @@ CRD_FILE=deploy/crds/jaegertracing.io_jaegers_crd.yaml
...
-
-git fetch -q upstream
-git checkout -q master
-git rebase -q upstream/master
+for dest in ${COMMUNITY_OPERATORS_REPOSITORY} ${UPSTREAM_REPOSITORY}; do
+    cd "${dest}"
+    git remote | grep upstream > /dev/null
+    if [[ $? != 0 ]]; then
+        echo "Cannot find a remote named 'upstream'. Adding one."
+        #git remote add upstream git@github.com:operator-framework/community-operators.git
```
Comment (jpkrohling) [inline .ci/operatorhub.sh:35]: Should this be commented out?

File: .ci/operatorhub.sh [hunk 2]
```diff
@@ -21,104 +27,45 @@ CRD_FILE=deploy/crds/jaegertracing.io_jaegers_crd.yaml
...
-
-<sup>2</sup> For more information see [here](https://github.com/operator-framework/operator-sdk/blob/master/doc/images/operator-capability-level.svg)
-EOM
-
+    tmpfile=$(mktemp /tmp/Update-Jaeger-to-${VERSION}.XXX)
+    echo  "Update Jaeger to v${VERSION}" > ${tmpfile}
+    cat  ${OLD_PWD}/.ci/.pr-template.md >> ${tmpfile}
     echo "Submitting PR on your behalf via 'hub'"
-    hub pull-request -F ${tmpfile}
+    gh pr create --title  "Update Jaeger to v${VERSION}" --body "$(cat ${tmpfile})"
```
Comment (jpkrohling) [inline .ci/operatorhub.sh:68]: You can use `--body-file` instead.

File: .ci/operatorhub.sh [hunk 3]
```diff
@@ -21,104 +27,45 @@ CRD_FILE=deploy/crds/jaegertracing.io_jaegers_crd.yaml
...
-
---
-
-<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need
-
-<sup>2</sup> For more information see [here](https://github.com/operator-framework/operator-sdk/blob/master/doc/images/operator-capability-level.svg)
-EOM
-
+    tmpfile=$(mktemp /tmp/Update-Jaeger-to-${VERSION}.XXX)
+    echo  "Update Jaeger to v${VERSION}" > ${tmpfile}
```
Comment (jpkrohling) [inline .ci/operatorhub.sh:65]: Do you need this? Can't we just send the template file as is?

Comment (rubenvp8510) [reply .ci/operatorhub.sh]: We can :) but I wanted to preserve the original behaviour. but yes we can, I will remove this.

File: .ci/operatorhub.sh [hunk 4]
```diff
@@ -21,104 +29,45 @@ CRD_FILE=deploy/crds/jaegertracing.io_jaegers_crd.yaml
...
-
---
-
-<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need
-
-<sup>2</sup> For more information see [here](https://github.com/operator-framework/operator-sdk/blob/master/doc/images/operator-capability-level.svg)
-EOM
+    tmpfile=$(mktemp /tmp/Update-Jaeger-to-${VERSION}.XXX)
+    echo "Update Jaeger to v${VERSION}" > ${tmpfile}
+    curl https://raw.githubusercontent.com/${dest}/main/docs/pull_request_template.md >> "${tmpfile}"
```
Comment (jpkrohling) [inline .ci/operatorhub.sh:67]: Sorry, what I meant in the previous comment is to use *our* template file. Our template file has items checked, like this:

```md
 * [x] A quadratic logo
```

I think we had a logic somewhere where we would check whether our template was still in sync with upstream, and fail the target if it isn't.

Comment (rubenvp8510) [reply .ci/operatorhub.sh]: Ok, I missunderstood your comment, 

Reintroduced the behaviour , now the check template is sending.

File: .ci/operatorhub [hunk 5]
```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+
+COMMUNITY_OPERATORS_REPOSITORY="k8s-operatorhub/community-operators"
```
Comment (jpkrohling) [inline .ci/operatorhub:4]: Is this now duplicated?

Comment (rubenvp8510) [reply .ci/operatorhub:4]: what do you mean?

**Comment by rubenvp8510** [comment]
@pavolloffay Could you review? Thanks!

File: .ci/.checked-pr-template.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+* [x] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
+* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
+* [x] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?
+
+### Updates to existing Operators
+
+* [x] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
+* [x] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
+* [x] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
+* [x] Have you tested an update to your Operator when deployed via OLM?
```
Comment (jpkrohling) [inline .ci/.checked-pr-template.md:18]: This was previously unchecked. Any reason for the change?

File: .ci/.checked-pr-template.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
+
+### Your submission should not
```
Comment (jpkrohling) [inline .ci/.checked-pr-template.md:21]: All those items were previously checked. Same with all other items from the other sections.

**Review by jpkrohling — COMMENTED** [review]
Everything looks good, just ensure the checkboxes are consistent with the state it was before if the change wasn't on purpose.

File: .ci/.checked-pr-template.md (hunk 7 above)
Comment (rubenvp8510) [reply .ci/.checked-pr-template.md:21]: Ah, I missunderstood this, you're right this should be checked.

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**